fastlane_core/lib/fastlane_core/configuration/config_item.rb

```diff
@@ -1,6 +1,6 @@
 module FastlaneCore
   class ConfigItem
-    attr_accessor :key, :env_name, :description, :short_option, :default_value, :verify_block, :optional, :conflicting_options, :conflict_block, :deprecated, :sensitive
+    attr_accessor :key, :env_name, :description, :short_option, :default_value, :verify_block, :optional, :conflicting_options, :conflict_block, :deprecated, :sensitive, :allow_shell_conversion
```

Comment:
Since there are so many now, would you mind splitting each attribute to a new line, and adding a comment above each of these? (you can copy those from below)